### PR TITLE
feat: landing-page [merged-develop-fix-code-review-issues]

### PR DESCRIPTION
**Pull request by Peter2221**
- add typography, btns, svg
- more less like figma design

### CODE REVIEW

File: src/domains/Landing/LandingPage/LandingPage.styled.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+export {};
```
Comment (Emu-Emax) [inline src/domains/Landing/LandingPage/LandingPage.styled.ts]: cut unecessary export

File: src/domains/Landing/LandingPage/LandingPage.styled.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export const MainTitle = styled.h1`
+    font-weight: 500;
```
Comment (Emu-Emax) [inline src/domains/Landing/LandingPage/LandingPage.styled.ts]: Please, use themed values, so : font-weight: ${({theme}) => theme.fontWeights.bold}, we have font-sizes, font-weight

File: src/shared/paths.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 export default {
     // dashboard
     dashboard: '/',
+    landing: '/landing',
```
Comment (Emu-Emax) [inline src/shared/paths.ts]: I think landing page would be our default route, so we can delete dashboard.

File: src/domains/index.ts [hunk 4]
```diff
@@ -1,2 +1,3 @@
 // dashboard components
 export { default as DashboardPage } from '@domains//Dashboard/DashboardPage/DashboardPage';
+export { default as LandingPage } from '@domains/Landing/LandingPage/LandingPage';
```
Comment (Emu-Emax) [inline src/domains/index.ts]: Please, fix prettier warnings

**Review by Wadowski — APPROVED** [review]

**Pull request merged**